.github/workflows/tests.yml

```diff
@@ -102,17 +107,15 @@ jobs:
 
       # Some workflows checkout WordPress from GitHub, but that seems to bring a bunch of uncompiled files with it.
       # Instead download from wordpress.org stable.
-      - name: Download WordPress
-        run: wget https://wordpress.org/wordpress-${{ matrix.wp-versions }}.tar.gz
-
-      - name: Extract WordPress
-        run: tar xfz wordpress-${{ matrix.wp-versions }}.tar.gz
-
-      # Checkout (copy) this repository's Plugin to this VM.
-      - name: Checkout Plugin
-        uses: actions/checkout@v4
-        with:
-          path: ${{ env.PLUGIN_DIR }}
+      - name: Download and Extract WordPress
```

Comment:
The change to using `/var/www/html` is because Ubuntu 22.x and higher with nginx 1.18.x and higher didn't work when using the old `/home/runner/work/convertkit-wordpress/convertkit-wordpress/wordpress` folder.